### ISSUE TITLE
Support for courses which offer classes on different weeks

### ISSUE THREAD

**Issue by catmanjan** [catmanjan/anutimetable#84]
This is an unlikely one since it would be a major change to the data source and scraper. Comment from Stalkerspace:

>  I like the alternative ANU timetable builder, however I can't seem to view the weeks that classes are on or off. Is there some extra function to do this, or should I dig through the ics file?

**Comment by samyex6** [comment]
..is that kind of data even show on the official timetable though?

**Comment by catmanjan** [comment]
Yeah it is, some classes have labs every second week and that is shown on the official timetable, haven't looked at that part of the datasource though so not sure how it is done.
